src/main/java/issame/achievements_core/mixin/MilkTossMixin.java

```diff
@@ -0,0 +1,30 @@
+package issame.achievements_core.mixin;
+
+import btw.entity.mob.villager.VillagerEntity;
+import issame.achievements_core.event.EventDispatcher;
+import net.minecraft.src.Entity;
+import net.minecraft.src.EntityPlayer;
+import net.minecraft.src.EntityVillager;
+import net.minecraft.src.World;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.Redirect;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+@Mixin(VillagerEntity.class)
+public class MilkTossMixin extends EntityVillager {
+    public MilkTossMixin(World par1World) {
+        super(par1World);
+    }
+
+    @Inject(method = "checkForLooseMilk",
```

Comment:
Change method to `checkForLooseMilk()V`.
Just want to be certain we're injecting into the right place